app/views/shared/_header.html.erb

```diff
@@ -41,7 +41,7 @@
           </li>
         <% end %>
         <li class="<%= 'active' if current_page?(controller: '/deploys', action: 'search') %>">
-          <%= link_to "Deploys", search_deploys_path %>
+          <%= link_to "Recent Deploys", search_deploys_path %>
```

Comment:
Is it just confusing because people are used to the old way or because it is actually confusing (to new people) ?
If it's the first I'd vote for keeping it ...